src/Adapters/EventSource.Net46/EventSource.Net46.csproj

```diff
@@ -0,0 +1,76 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Import Project="..\..\..\..\packages\MicroBuild.Core.0.2.0\build\MicroBuild.Core.props" Condition="Exists('..\..\..\..\packages\MicroBuild.Core.0.2.0\build\MicroBuild.Core.props')" />
+  
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{A539BB1D-29FF-48E8-8D6E-DFCC543DC2B4}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Microsoft.ApplicationInsights.EventSourceListener</RootNamespace>
+    <AssemblyName>Microsoft.ApplicationInsights.EventSourceListener</AssemblyName>
+    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
```

Comment:
No. There is an important API that is missing from net45 [EventWrittenEventArgs.PayloadNames](https://msdn.microsoft.com/en-us/library/system.diagnostics.tracing.eventwritteneventargs.payloadnames(v=vs.110).aspx). When I was working on an earlier version of this code, I had long discussions with Vance about possible workarounds. Ultimately we decided it is not worth the effort. Now that 4.6 is installed by default on Windows Azure VMs, there is even less incentive to support 4.5.